.npmignore

```diff
@@ -1 +1,10 @@
-logo
+.editorconfig
+.eslintcache
+.eslintignore
+.eslintrc.json
+.github
+.lintstagedrc
+assets
```

Comment:
Yep fair point, I'll have a re-think, could we get away with storing it in the documentation folder?